code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -777,3 +759,146 @@ Assign your candidates in choose_candidates() instead.
+
+	//and now we manipulate Dynamic to force a bunch of other rulesets. It just works...
+	var/list/the_madness = list(
+		"Malfunctioning AI",
+		"Blob Conglomerate",
+		"Blood Cult",
+		"Nuclear Emergency",
+		"Wizard",
+		"Revolution",
+		"Vampires",
+		"Changelings",
+		"Syndicate Traitors",
+		)//The order matters. As a rule of thumb, rarer rulesets should have their chance come first, while more common ones will fill the spots
+
+	if (assigned.len < 5)
+		the_madness -= "Revolution"//No revs unless there's at least 5 roundstart heads, because that'd be overkill
+
+	for (var/madness_ruleset in the_madness)
+		for (var/datum/dynamic_ruleset/roundstart/rule in mode.roundstart_rules)
+			if (madness_ruleset == rule.name)
+				rule.candidates = candidates.Copy()
+				rule.trim_candidates()
+				if (rule.ready(TRUE))
+					rule.calledBy = "antag madness"
+					rule.required_candidates = 1//because we're funny
+					if (madness_ruleset == "Syndicate Traitors")//no traitor limits
+						rule.protected_from_jobs = list()
+						rule.restricted_from_jobs = list("Mobile MMI")
+						for (var/mob/M in candidates)
+							rule.assigned += M
+							rule.candidates -= M
+					else
+						rule.choose_candidates()
+					if (rule.assigned.len)
+						message_admins("ANTAG MADNESS: <font size='3'>[rule.name]</font> OK!")
+						log_admin("ANTAG MADNESS: <font size='3'>[rule.name]</font> OK!")
+						mode.threat_log += "[worldtime2text()]: Roundstart [rule.name] forced by antag madness"
+
+						if (istype(rule, /datum/dynamic_ruleset/roundstart/delayed/))
+							message_admins("ANTAG MADNESS: with a delay of [rule:delay/10] seconds.")
+							log_admin("ANTAG MADNESS: with a delay of [rule:delay/10] seconds.")
+							mode.pick_delay(rule)
+
+						if (rule.execute())
+							rule.stillborn = IsRoundAboutToEnd()
+							mode.executed_rules += rule
+							if (rule.persistent)
+								mode.current_rules += rule
+							for(var/mob/M in rule.assigned)
+								candidates -= M
+					else
+						message_admins("ANTAG MADNESS: <font size='3'>[rule.name]</font> FAILED!")
+						log_admin("ANTAG MADNESS: <font size='3'>[rule.name]</font> FAILED!")
+				else
+					message_admins("ANTAG MADNESS: <font size='3'>[rule.name]</font> FAILED!")
+					log_admin("ANTAG MADNESS: <font size='3'>[rule.name]</font> FAILED!")
+
+	spawn(10)
+		message_admins("<span class='danger'>Antag Madness is now underway. In this very chaotic mode, admins are encouraged to proactively interfere with the round to keep things interesting, and help it reach a conclusion if necessary. Blobs and Rev may no longer cause the round to suddenly end, Malf and Nuke Ops still can if they destroy the station. If things get stale after 30 minutes and there are lots of dead players, consider bringing the round to a conclusion by either forcing a shuttle call, sending either the deathsquad or elite syndies to destroy the station, or trigger a supermatter cascade (or combination of those).</span>")
```

Comment:
I realize that having blob win might be anticlimactic, but it's still weird that neither revs or blob can win at all (hardcoded) and admins can't even bus them to win because it's written into their winstates. If these factions are winning you MUST bus their failure or you end up with a dead husk of a station where the blob has killed everything but it's not ending.

Reply:
I guess it'd take at least 30 minutes for blob to win anyway so I'll backtrack on that and no longer have the mode prevent blob victory. I think it is warranted for revs still however. Admins can still set the ticker's revolutionary_victory var to 1 if they want to enable castle voting.